app/config/Module.scala

```diff
@@ -28,6 +28,7 @@ class Module extends AbstractModule {
     bind(classOf[Clock]).toInstance(Clock.systemDefaultZone.withZone(ZoneOffset.UTC))
     bind(classOf[JourneyStateRepository]).to(classOf[MongoJourneyStateRepository])
     bind(classOf[JourneyAnswersRepository]).to(classOf[MongoJourneyAnswersRepository])
+    ()
```

Comment:
Does our compiler now warn us (now = build error) if we do not have `()` as the last line of a method that evaluates to unit?

Reply:
I think the warning is by default, that you're returning different type that declared. I just added that it will fail build, so developer can go back to the line and make concious choise (changing the return type, or returning unit).

Here the bind and to return non unit value so that's why you need to add (). But if Unit function call another Unit function you don't need to use (I)